### ISSUE TITLE
Learning rate and other hyper-parameters to train OpenCLIP on LAION 400M

### ISSUE THREAD

**Issue by LiqunChen0606** [mlfoundations/open_clip#100]
Hi there,

Thanks very much for the work! I wonder if you guys can share the learning rate (along with other hyper-parameters) to train CLIP on LAION 400M dataset? If you have some special pipeline to train, could you also help to share with us? 

Thanks in advance!

**Comment by Zasder3** [comment]
Hi Liqun!

We use the exact hyperparameters listed in the CLIP paper (available on the last page). Surprisingly no special pipeline! We use this exact codebase and the WebDataset integration as our data format.

Hope this helps :)
